pom.xml

```diff
@@ -15,7 +15,8 @@
     <contact.address>geoevent@esri.com</contact.address>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.bundle.plugin.version>4.2.1</maven.bundle.plugin.version>
-    <activemq.version>5.16.0</activemq.version>
+    <activemq.version>5.16.4</activemq.version>
+    <!-- can't go to 5.17 if GeoEvent is running under JRE8 - which 10.8.x and earlier do -->
```

Comment:
can you rename `GeoEvent` to `ArcGIS GeoEvent Server` in the comment here?
`GeoEvent` is the actual event (message), not the official product name.

Reply:
OK